tests/integration/store/test_pandas_store.py

```diff
@@ -797,3 +796,26 @@ def test_daterange_fails_with_timezone_start(library):
     with pytest.raises(ValueError):
         library.read('MYARR', date_range=DateRange(start=dt(2015, 1, 1, tzinfo=mktz())))
 
+def test_data_info_series(library):
+    s = Series(data=[1, 2, 3], index=[4, 5, 6])
+    library.write('pandas', s)
+    md = library.read('pandas').data_info
+    assert md == library.read_metadata('pandas').data_info
+    assert md == {'col_dtypes': "[('index', '<i8'), ('values', '<i8')]", 'col_names': {'columns': ['values'], 'index': ['index']}, 'type': 'pandasseries'}
```

Comment:
Would it be useful for the `col_dtypes` to be a list of tuples (rather than a string __repr__)?